emgapianns/management/commands/import_analysis_summaries.py

```diff
@@ -0,0 +1,34 @@
+from django.core.management.base import BaseCommand
+from emgapi.models import AnalysisJob
+
+
+class Command(BaseCommand):
+    help = 'Copy values from analysis_summary to analysis_summary_json for a specified batch of AnalysisJob records'
+
+    def add_arguments(self, parser):
+        parser.add_argument('batch_number', type=int, help='Batch number to process')
+
+    def handle(self, *args, **options):
+        batch_number = options['batch_number']
+        batch_size = 10000  # Set your desired batch size here
+
+        try:
+            # Calculate the starting and ending index for the batch
+            start_index = (batch_number - 1) * batch_size
+            end_index = batch_number * batch_size
+
+            # Get AnalysisJob records for the specified batch
+            analysis_jobs = AnalysisJob.objects.all()[start_index:end_index]
+
+            # Print the number of records in the batch
+            self.stdout.write(self.style.SUCCESS(f'Processing batch {batch_number} of {len(analysis_jobs)} records.'))
+
+            for analysis_job in analysis_jobs:
```

Comment:
Could we use [bulk_update](https://docs.djangoproject.com/en/3.2/ref/models/querysets/#bulk-update) ? This lets you write a column to many rows in one DB transaction, so should speed things up quite considerably when we run it for real (because the DB is over network).

Reply:
@SandyRogers I;ve added bulk update